From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Yocto <yocto@local>
Date: Mon, 8 Sep 2025 13:55:00 +0000
Subject: [PATCH] disable pulseaudio backend check (force off)

Avoid failing configure due to missing libpulse when building with
--with-backends=alsa. Force BUILD_PULSEAUDIO=false irrespective of
detected packages or options.

---
 wscript | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/wscript b/wscript
index 1111111..2222222 100644
--- a/wscript
+++ b/wscript
@@ -1346,7 +1346,7 @@ def configure(conf):
     conf.env['BUILD_DUMMYBACKEND'] = any('dummy' in b for b in backends)
     conf.env['BUILD_PABACKEND'] = any('portaudio' in b for b in backends)
     conf.env['BUILD_CORECRAPPITA'] = any('coreaudio' in b for b in backends)
-    conf.env['BUILD_PULSEAUDIO'] = any('pulseaudio' in b for b in backends)
+    conf.env['BUILD_PULSEAUDIO'] = False
 
     if backends == [''] or not (
                conf.env['BUILD_JACKBACKEND']
             or conf.env['BUILD_ALSABACKEND']
@@ -1378,7 +1378,7 @@ def configure(conf):
     if re.search ("linux", sys.platform) == None and conf.env['BUILD_PULSEAUDIO']:
         conf.fatal("Pulseaudio Backend is only available on Linux")
 
-    if conf.env['BUILD_PULSEAUDIO'] and not conf.is_defined('HAVE_PULSEAUDIO'):
+    if False and conf.env['BUILD_PULSEAUDIO'] and not conf.is_defined('HAVE_PULSEAUDIO'):
         conf.fatal("Pulseaudio Backend requires libpulse-dev")
 
     set_compiler_flags (conf, Options.options)
-- 
2.34.1

